### PR TITLE
Initial folder organizations and dependencies

### PR DESCRIPTION
**Pull request by ACRUNNER**
Not sure if these are the right dependencies to use

### CODE REVIEW

**Comment by ACRUNNER** [comment]
Hi! I created a gitignore and removed the node modules. This can now start a a server and has a basic front end to show its working.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+node_modules
+public/bundle.js
+package-lock.json
```
Comment (frokrab) [inline .gitignore]: You should include the package-lock.json file in your commits; it makes sure that other people who use your code will get the right version of each dependency when they run npm install on their own computers.

File: client/src/index.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+var React = require('react');
+var ReactDOM = require('react-dom');
+
+class App extends React.Component{
```
Comment (frokrab) [inline client/src/index.jsx]: Make sure the indentations in this component are all 2 spaces

File: index.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+var express = require('express');
+var bodyParser = require('body-parser');
+
+var app = express();
+app.set('port', 3003);
+app.use(bodyParser.json());
+app.use(express.static(__dirname + '/public/'));
+app.listen(app.get('port'));
```
Comment (frokrab) [inline index.js]: If you just use app.listen(3003) here it'll save you from having to set the port on line 5

File: public/index.html [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<!doctype html>
+	<head>
+		<title>FEC</title>
+	</head>
+	<body>
+		<div id = "app"></div>
+		<script src = "bundle.js"></script>
+	</body>
```
Comment (frokrab) [inline public/index.html]: Watch the spacing on your tabs again here! Also I think head and body should be flush with the doctype tag

**Comment by ACRUNNER** [comment]
Hey Nick,
Thanks for the help. Did changes.
Anton

**Pull request merged**